### DISCUSSION TITLE
Support for ESP32-C2

### DISCUSSION THREAD

**Discussion by XanCraft21** [espressif/arduino-esp32#7786] [Ideas]
Hello, i was wondering if there are any plans to add support for ESP32-C2 for Arduino. I have seen the boards online to buy but no support at this time. I have searched for other discussions and it seems like no-one has posted anything about it yet, and have not found any other info on the support plans. Is the ESP32-C2 a real chip or is it a clone chip from china? Are there any plans to add support to the Arduino package? Thank you for answering my questions.

**Comment by pedrominatel** [comment]
Hi,

Right now we are focusing on the upgrade to ESP-IDF 5.1, which will include support not only for the ESP32-C2 but for the ESP32-C6. After that, we will work on the new chips support, but not before the ESP-IDF upgrade.

You can follow the roadmap here: https://github.com/orgs/espressif/projects/3

**Comment by XanCraft21** [comment]
Thank you. I look forward to this update. I’ve been waiting to buy an ESP32-C2 for a while. I have not seen the ESP32-C6 out online yet on the shopping site i use.

EDIT: I just look on AliExpress on the official seller and they have one ESP32-C6 out. So far no resellers have it yet. I plan to buy it from the official store when the support comes out.

**Comment by jezmck** [comment]
I have a "NodeMCU ESP32-C2 (ESP8684)", it it usable?

I've had no luck trying to use it with Arduino IDE nor PlatformIO.

**Reply by me-no-dev** [reply]
I'm pretty sure you can use it with pioarduino cc @Jason2866

**Comment by Jason2866** [comment]
@jezmck Since you already know Platformio it is very easy, just move to my fork [pioarduino](https://github.com/pioarduino/platform-espressif32) and you are ready to go.
